webpack.config.js

```diff
@@ -211,6 +213,21 @@ module.exports = {
 				),
 				to: 'build/block-library/blocks/[1]/block.json',
 			},
+			{
+				from: './packages/edit-widgets/src/blocks/**/index.php',
+				test: new RegExp(
+					`([\\w-]+)${ escapeRegExp( sep ) }index\\.php$`
+				),
+				to: 'build/edit-widgets/blocks/[1].php',
+				transform: transformBlockContent,
+			},
+			{
+				from: './packages/edit-widgets/src/blocks/*/block.json',
+				test: new RegExp(
+					`([\\w-]+)${ escapeRegExp( sep ) }block\\.json$`
+				),
+				to: 'build/edit-widgets/blocks/[1]/block.json',
+			},
```

Comment:
Seems like we could simplify this a bit further by having an array of paths where block could live and generate this with a loop?

Reply:
Good call!